Dockerfile_build_ubuntu64

```diff
@@ -36,7 +36,7 @@ RUN gem install fpm
 # Install protobuf3 protoc binary
 ENV PROTO_VERSION 3.11.1
 RUN wget -q https://github.com/google/protobuf/releases/download/v${PROTO_VERSION}/protoc-${PROTO_VERSION}-linux-x86_64.zip \
-    && unzip -j protoc-${PROTO_VERSION}-linux-x86_64.zip bin/protoc -d /bin \
+    && unzip -o -j protoc-${PROTO_VERSION}-linux-x86_64.zip bin/protoc -d /bin \
```

Comment:
`-o` needed to overwrite the `protoc` binary that's included in the x-builder image. We could instead delete this whole block from the dockerfile here, but I was wary about switching from the pinned version here to whatever happens to get pulled from the apt repos.

Reply:
I'm not too worried if we pin the build - it should be consistent since we pull the same cross build container every time now.

If we still want to pin it, why not in the parent image?

Reply:
btw from the protobuf docs:

```
A new notion "syntax" is introduced to specify whether a .proto file
uses proto2 or proto3:

// foo.proto
syntax = "proto3";
message Bar {...}
If omitted, the protocol buffer compiler generates a warning and "proto2" is
used as the default. This warning will be turned into an error in a future
release.
```

Hopefully we use `syntax` everywhere but if we don't then it is still backward-compatible with newer versions of the protobuf compiler.

Reply:
I'll just use the version from the cross-builder

Reply:
Ah wait, that is if it includes the python libraries... need to check

Reply:
Scratch that, will update the cross-builder image to use the version pinned here & install the matching c++ runtime. This image will still have access to the PROTO_VERSION env var from the cross-builder, so it can use that to also download & install the python runtime.